.travis.yml

```diff
@@ -2,9 +2,28 @@ language: python
 sudo: false
 python:
     - "2.7"
-install: 
+install:
     - pip install -r requirements.txt
     - npm install -g grunt-cli bower
 script:
     - python manage.py migrate --fake-initial
     - python manage.py compile_frontend
+
+# Simple smoke tests
+after_deploy:
+  - curl --silent --fail --head https://fec-eregs-dev.apps.cloud.gov || travis_terminate $?
+  - curl --silent --fail --get https://fec-eregs-dev.apps.cloud.gov/api/regulation/2 > /dev/null || travis_terminate $?
```

Comment:
Also, just for readability, I would put the `after_deploy` below the `deploy` section.